src/main/java/org/gridsuite/dynamicsecurityanalysis/server/entities/DynamicSecurityAnalysisResultEntity.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.dynamicsecurityanalysis.server.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.gridsuite.dynamicsecurityanalysis.server.dto.DynamicSecurityAnalysisStatus;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@Getter
+@Setter
+@Table(name = "dynamic_security_analysis_result")
+@NoArgsConstructor
+@Entity
+public class DynamicSecurityAnalysisResultEntity implements Serializable {
```

Comment:
you need " implements Serializable" ?

Reply:
YES! I don't see any case in which we need Serializable, this entity has no relation with others and normally endpoints do not use directly an entity. 

I will remove Serializable

Reply:
DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c